### DISCUSSION TITLE
Invited guests not saved and Invites not being sent out

### DISCUSSION THREAD

**Discussion by MedigateIT** [kewisch/gdata-provider#299] [Support] [answered]
Hello, I've been using this add-on for thunderbird for a while now, and after I had to diagnose another issue with add-ons I had to disable it. After re-enabling it, it works ok, but I noticed that invites to  events created in thunderbird are not sent.
When I try to go to add-on settings and change anything there, it does not save what I choose.

I tried searching for a similar issue, but found none.
thanks.

**Comment by kewisch** [comment]
Can you check the add-on options? There are some new checkboxes that should help you control the settings.

**Reply by MedigateIT** [reply]
You mean these:
![image](https://user-images.githubusercontent.com/93334657/142752356-78a7d8a1-b515-432b-a503-8b43daa35932.png)
these are the ones that do not save, I ticked them and when I close the page and reopen it they're unticked again.

**Reply by raymondjpg** [reply]
I have the same issue.  Add-on options checked, Thunderbird re-started, options all show unchecked again.

**Reply by web-elements-ch** [reply]
We have the exact same problem. The options are not being saved.

**Reply by nldasha** [reply]
Same issue here. Options will not stay checked and cannot invite attendees

**Reply by GeorgiAngelovStangaOne** [reply]
Same issue here, options not saving and no invitations sent. Is there any news about this problem ?

**Reply by kewisch** [reply]
Yes, fixed this in #312 and will be releasing soon

**Comment by kewisch** [comment, marked as answer]
Elevating this to an answer, I've fixed this in #312. Thank you for reporting!
